ratis-server/src/main/java/org/apache/ratis/server/storage/SnapshotManager.java

```diff
@@ -131,4 +133,12 @@ public void installSnapshot(StateMachine stateMachine,
       tmpDir.renameTo(dir.getStateMachineDir());
     }
   }
+
+  // Rename a file by appending .corrupt to file name. This function does not guarantee
+  // that the rename operation is successful.
+  @VisibleForTesting
+  static void renameFileToCorrupt(File tmpSnapshotFile) {
```

Comment:
why static ?

Reply:
Because this function does not access any class member. Any suggestion or style guide that says better to not use static?

Reply:
I think it should be a common util, and should not placed here.

Reply:
Thanks for your reminder.

I didn't find such function implemented in common util, but indeed I found something useful so I pushed to adopt those. I also moved the rename-to-corrupt to FileUtils.